src/pages/index.tsx

```diff
@@ -40,35 +40,44 @@ export const getStaticProps: GetStaticProps = async () => {
 
 type Props = {
   bubbles: (Bubble & {
+    labels: [];
     author: {
         avatarUrl: string;
     };
   })[]
 };
 
 const HomePage: React.FC<Props> = (props: Props) => {
-  const [modalIsVisible, setModalIsVisible] = useState(false)
+  const [bubbleDetails, setBubbleDetails] = useState(false)
```

Comment:
Hmmm assim da a impressão que bubble details é um boolean.
Pode deixar assim, mas só pra readability das próximas coloca algo tipo:
isBubbleDetailsVisible. (sempre começa com "is" pra boolean). js temos que tomar cuidado com readability por nao ter tipos

Reply:
Blz. Arrumei la

Reply:
em outra branch? tem que commitar aqui man kk